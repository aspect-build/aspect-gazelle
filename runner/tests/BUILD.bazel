load("@aspect_orion//:gazelle.bzl", "gazelle_generation_test")
load("@bazel_gazelle//:def.bzl", "gazelle_binary")

# Disable the gazelle extensions
# gazelle:exclude .

gazelle_binary(
    name = "gazelle_bin",
    testonly = True,
    languages = [
        "@aspect_orion//language/js",
        "//git/testing",
    ],
    visibility = ["//visibility:private"],
)

[
    gazelle_generation_test(
        name = "%s_test" % t,
        dir = t,
        gazelle_binary = ":gazelle_bin",
    )
    for t in [t.replace("/WORKSPACE", "") for t in glob(["*/WORKSPACE"])]
]
