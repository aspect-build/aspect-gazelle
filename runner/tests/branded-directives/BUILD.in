# aspect:generation_mode update_only
# aspect:map_kind ts_project ts_my_project //:defs.bzl
