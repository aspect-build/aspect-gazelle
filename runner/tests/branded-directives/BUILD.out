load("//:defs.bzl", "ts_my_project")

# aspect:generation_mode update_only
# aspect:map_kind ts_project ts_my_project //:defs.bzl

ts_my_project(
    name = "branded-directives",
    srcs = ["sub/x.ts"],
)
